src/app/gantt-chart/gantt-chart.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component } from '@angular/core';
+import * as Highcharts from "highcharts/highcharts-gantt";
+import ExportingModule from 'highcharts/modules/exporting';
```

Comment:
If the same instance would be used for all cards then there will be no need to install exporting in each chart component.